vernac/g_vernac.mlg

```diff
@@ -197,6 +197,12 @@ END
 let warn_plural_command =
   CWarnings.create ~name:"plural-command" ~category:CWarnings.CoreCategories.pedantic ~default:CWarnings.Disabled
          (fun kwd -> strbrk (Printf.sprintf "Command \"%s\" expects more than one assumption." kwd))
+let warn_plural_command ?loc k =
+  match loc with
+  | None -> warn_plural_command k
+  | Some loc ->
+      let kwd = Str.replace_first (Str.regexp "s$") "" k in
```

Comment:
that's not correct is it? on `Hypotheses` it will produce `Hypothese` instead of `Hypothesis`

Reply:
good catch! I'll remove this last commit from the PR, there is also a TODO with a wrong loc that I don't have the time to fix